app/models/curator/metastreams/descriptive.rb

```diff
@@ -7,7 +7,13 @@ class Metastreams::Descriptive < ApplicationRecord
     include AttrJson::Record::Dirty
     include AttrJson::NestedAttributes
 
-    enum digital_origin: ['born digital', 'reformatted digital', 'digitized microfilm', 'digitized other analog'].freeze
+    enum digital_origin: {
```

Comment:
If the keys and values are the same, could this not be rewritten as:
```ruby
enum digital_origin: %w(born_digital reformatted_digital digitized_microfilm digitized_other_analog).freeze
```
Or can we keep the space in the values, but not the keys? As in:
```ruby
born_digital: 'born digital'
```

Reply:
We should keep them underscored. I can try to change it to an array